build/Targets/VSL.Imports.targets

```diff
@@ -132,6 +132,26 @@
         <MaxVersion>$(VisualStudioReferenceAssemblyVersion)</MaxVersion>
       </SuggestedBindingRedirects>
   </ItemGroup>
+
+  <ItemGroup Condition="'$(GenerateCompilerBindingRedirects)' == 'true'">
```

Comment:
Can we move this to a sepate .targets file imported by project that need it? VSL.Imports is becoming a kitchen sink of stuff.